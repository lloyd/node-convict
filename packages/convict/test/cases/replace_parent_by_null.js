'use strict';

exports.conf = {
  first: {
    second: {
      third: {
        four: {
          default: 4,
          format: 'int'
        }
      }
    }
  }
};
