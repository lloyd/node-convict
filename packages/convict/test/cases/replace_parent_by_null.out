Validate failed because wrong value(s):
  - config parameter "first.second.third.four" missing from config, did you override its parent? Because "first.second.third" is null.